sql/core/src/test/resources/sql-tests/results/interval.sql.out

```diff
@@ -1,5 +1,5 @@
 -- Automatically generated by SQLQueryTestSuite
```

Comment:
not related to this PR: we should put the test queries that have different behaviors if ansi mode is on to a different file, and only test that file with ansi mode on and off.